backend/gateway/config.go

```diff
@@ -28,13 +28,15 @@ type Flags struct {
 	ConfigPath string
 	Template   bool
 	Validate   bool
+	DevMode    bool
 }
 
 // Link register the struct vars globally for parsing by the flag library.
 func (f *Flags) Link() {
 	flag.StringVar(&f.ConfigPath, "c", "clutch-config.yaml", "path to YAML configuration")
 	flag.BoolVar(&f.Template, "template", false, "executes go templates on the configuration file")
 	flag.BoolVar(&f.Validate, "validate", false, "validates the configuration file and exits")
+	flag.BoolVar(&f.DevMode, "dev", false, "run clutch in development mode")
```

Comment:
this is confusing to me because dev mode usually implies hot restart and other things, (e.g. our target `make backend-dev`)

should we just thread through the specific filename of the env file? that would work better for keeping random env files in examples directories and such too.